kaggle/src/connector.js

```diff
@@ -0,0 +1,291 @@
+var cacheDuration = 60 * 60 & 6;
```

Comment:
Should the `&` be a `*`? If not, please add a comment explaining what this does.

Reply:
I still don't follow. If we want the cache duration to be 0, why wouldn't we set it directly to be zero? If we want it to be 6 hours, it needs to be `60 * 60 * 6` instead of `60 * 60 & 6`